tests/testthat/test_smaller.R

```diff
@@ -0,0 +1,320 @@
+## Tests for object creation and preprocessing
+
+set.seed(1)
+
+pbmc.file <- system.file('tests', 'testdata', 'smaller_pbmc_data.RDS', package = 'liger')
+pbmc.small <- readRDS(pbmc.file)
+
+# Tests for object creation 
+####################################################################################
+context("Object creation")
+
+ligex <- createLiger(raw.data = pbmc.small, make.sparse = T, take.gene.union = F,
+                     remove.missing = T)
+test_that("Object instantiation creates liger object", {
+  expect_is(ligex, "liger")
+})
+
+test_that("Dataset names passed correctly", {
+  expect_identical(names(ligex@raw.data), c('tenx', 'seqwell'))
+})
+
+test_that("Sparse matrices created", {
+  expect_is(ligex@raw.data[[1]], "CsparseMatrix")
+})
+
+# is cell.data still a thing?
```

Comment:
Sometimes your session has loaded a previous version of liger and re-installing doesn't automatically reload the package. If you see things like this happening, I would recommend restarting your R session and then reloading the package.